mmdet3d/apis/test.py

```diff
@@ -44,7 +44,9 @@ def single_gpu_test(model,
             models_3d = (Base3DDetector, Base3DSegmentor,
                          SingleStageMono3DDetector)
             if isinstance(model.module, models_3d):
-                model.module.show_results(data, result, out_dir=out_dir)
+                gt_bboxes = data_loader.dataset.get_ann_info(i)['gt_bboxes_3d']
```

Comment:
Please assign `gt_bboxes = None` if the dataset doesn't have `gt_bboxes_3d` annotation. Can be something like
```
if data_loader.dataset.test_mode:
    gt_bboxes = None
```
See e.g. this [line](https://github.com/open-mmlab/mmdetection3d/blob/e5a87f35ad65c2a2c31bf6888293bfef1b963bed/mmdet3d/datasets/kitti_dataset.py#L133)

Reply:
Another problem is that, if the model being tested is a 3D segmentor or a multi-modality detector. May need to deal with different situations here (e.g. for segmentor, its GT is `pts_semantic_mask`)